azure/converters/vmss_test.go

```diff
@@ -122,3 +123,137 @@ func Test_SDKToVMSS(t *testing.T) {
 		})
 	}
 }
+
+func Test_SDKToVMSSVM(t *testing.T) {
```

Comment:
Would it be helpful to add a test case for a nil `VirtualMachineScaleSetVMProperties`?

Reply:
Good idea–we don't explicitly set it to `nil` in the tests, but that is a branch in the function being tested. Added.